dash/backend/src/modules/kube-hunter/dto/kube-hunter-raw.interface.ts

```diff
@@ -0,0 +1,90 @@
+export interface IKubeHunterRawNodes extends IKubeHunterRawResponseSection {
+  value: Nodes;
+}
+export interface IKubeHunterRawServices extends IKubeHunterRawResponseSection {
+  value: Services;
+}
+export interface IKubeHunterRawVulnerabilities extends IKubeHunterRawResponseSection {
+  value: Vulnerabilities;
+}
+
+interface IKubeHunterRawResponseSection {
+  value: Nodes | Services | Vulnerabilities;
+  key: string;
+  obj: RawObj;
+  type: number;
+  options: Options;
+}
+
+interface Nodes {
+  value: Node[]
+  key: string
+  obj: ObjDefinition
+  type: number
+  options: Options
+}
+
+interface Services {
+  value: Service[]
+  key: string
+  obj: ObjDefinition
+  type: number
+  options: Options
+}
+
+interface Vulnerabilities {
+  value: Vulnerability[]
+  key: string
+  obj: ObjDefinition
+  type: number
+  options: Options
+}
+
+interface ObjDefinition {
+  nodes: Node[]
+  services: Service[]
+  vulnerabilities: Vulnerability[]
+  uuid: string
+  clusterId: number
+}
+
+interface Node {
+  type: string
+  location: string
+}
+
+interface Service {
+  service: string
+  location: string
+}
+
+interface Vulnerability {
+  location: string
+  vid: string
+  category: string
+  severity: string
+  vulnerability: string
+  description: string
+  evidence: string
+  avd_reference: string
+  hunter: string
+}
+
+interface Options {
+  enableCircularCheck: boolean
+  enableImplicitConversion: boolean
+  excludeExtraneousValues: boolean
+  exposeDefaultValues: boolean
+  exposeUnsetFields: boolean
+  ignoreDecorators: boolean
+}
```

Comment:
A lot of the interfaces in this file are very generically named. It would probably be a good idea to name them something like KubeHunterWhatever.

Right now all the generic ones are just used in IKubeHunterRawResponseSection, but in the future, if we have a reason to do more processing on the reports, we may need to export these inner interface

Reply:
So I generated the interfaces by throwing the response into an interface generator ;)
I had made the names slightly more explicit and consolidated repeat interfaces, but I hadn't bothered making them too clear since they were internal interfaces.
The latest commit adds `IKHRawResponse` to the beginning of each interface.